src/ui/popup.js

```diff
@@ -231,6 +234,22 @@ export default class Popup extends Evented {
         return this.setDOMContent(frag);
     }
 
+    /**
+     * Sets the popup's max width.
+     *
+     * @param pixel A string representing the pixel value for the maximum width.
```

Comment:
The name of the parameter here in the doc should be the same as the argument to the method, which in this case is `maxWidth`. I'd also make it clear that this is setting the CSS property `maxWidth` and clarify that the expected string format is "Npx" where N is some number.